test/feeder_ex_test.exs

```diff
@@ -30,6 +30,11 @@ defmodule FeederExTest do
 
   end
 
+  test "parse! returns just the parsed results on success" do
+    parsed_feed = FeederEx.parse!(File.read!(@sample_file))
+    assert parsed_feed.title == "Liftoff News"
+  end
```

Comment:
I think I'm misunderstanding something here. I tried the above (the replaced `</xml>`) and it still happily parsed the result. So I'm still not quite sure how to create the failing test. 

Anyway, I'll push up the spec I made so far.